testing/test-app/tests/rendering/aria-grid-test.ts

```diff
@@ -126,14 +131,18 @@ module('{{aria-grid}}', function (hooks) {
   test('Only the first cell has tabindex=0', async function (assert) {
     await render(hbs`<Grid />`);
 
-    assert.dom(selectors.tabbable).exists({ count: 1 });
+    assert.dom(gridSelectors.tabbable).exists({ count: 1 });
+
+    await render(hbs`<Table />`);
```

Comment:
1. I made a separate file to test the `<Table />` and duplicated all the grid tests into it.
2. This was probably a good thing because I didn't test the keyboard navigation and caught a bunch of bugs.
3. I fixed those and got tests passing locally, but it's now failing in CI. 

It's as if the build online is missing these commits:
https://github.com/CrowdStrike/ember-aria/pull/23/commits/5f09b434337cb8ab2773acf4bb1f086b5b21dd94
https://github.com/CrowdStrike/ember-aria/pull/23/commits/04fcbc0f6c3ed4dbb694602d00d262884e8a4c60

4. Locally, I usually do
```
pnpm i 
pnpm build 
pnpm i -f // re-symlink (I seem to need -f to force, but other people do not)
```

But it looks like assert-build only builds the js? https://github.com/CrowdStrike/ember-aria/blob/main/.github/actions/assert-build/action.yml#L12

Reply:
I wonder if you need to add this: https://github.com/CrowdStrike/ember-headless-table/blob/main/.github/actions/download-built-package/action.yml#L11-L13
(since we've been using peers more, everything that runs in to this pnpm<->peer bug needs to re-install after downloading the assets)

Reply:
Looks like we have a variation of that already: https://github.com/CrowdStrike/ember-aria/blob/main/.github/actions/download-built-package/action.yml#L11-L13

Reply:
ah I see -- also _most_ of the tests are passing, so it's likely not related to the symlink issue.

can you try locally recursively deleting all your node_modules -- and your pnpm-lock.json, and see if things start reproducing locally?

Reply:
Will do.

Also that table template might need the related helpers...I notice the `macos` ones seem to be failing. So I will try adding those in also.